### ISSUE TITLE
Tests hidden in VS Code's 'Explorer' cannot be found

### ISSUE THREAD

**Issue by nils-soderman** [microsoft/vscode-extension-test-runner#44]
To keep my VS Code explorer clean, I tend to exclude _out_ & _dist_ folders using the `files.exclude` setting. This however, leads to this extension failing to locate the tests as it relies on `vscode.workspace.findFiles`. 

Maybe `glob` can be used instead to locate all tests on the filesystem?

Cheers,
Nils

**Comment by connor4312** [comment]
@andreamah is this something I can use the new `findFiles2` API to fix? Sorry I lost track of where all the proposals are 🙈

**Comment by andreamah** [comment]
Yes, you can use the new findFiles2 for this! You can use the existing shape for `findFiles2` to achieve this right now, but I'd try to be ready to move to the new shape once I introduce it.
